### PR TITLE
test(coap-server): don't shutdown servient unnecessarily

### PR DESCRIPTION
**Pull request by JKRhb**
This PR contains a potential fix for the CoAP server problems observed in #1195. As the servients are not used by the `CoapServer` class, I simply removed the invocations of `shutdown` to fix the issue at hand.

In general, I think we could discuss if passing the servient to the protocol interface implementations is actually necessary or if the passing of protocol-specific callbacks for retrieving credentials could be an alternative (this is the approach I used for dart_wot in the meantime). But I would open a dedicated issue for that later :)

### CODE REVIEW

**Review by relu91 — APPROVED** [review]
Good to go too, should we merge this first or maybe try it after we merged #1195 ?

**Comment by JKRhb** [comment]
> Good to go too, should we merge this first or maybe try it after we merged #1195 ?

Either way would be fine for me :) If we merge #1195 first, then I would rebase against the new `master` state.

**Review by danielpeintner — APPROVED** [review]

**Comment by danielpeintner** [comment]
I would merge this PR before https://github.com/eclipse-thingweb/node-wot/pull/1195 so that we have clean runs for all PRs that we merge. I hope this is okay for all..

**Pull request merged**
